.travis.yml

```diff
@@ -1,71 +1,47 @@
 language: python
+cache: pip
+
 python:
-  - "2.6"
   - "2.7"
-  - "3.4"
-  - "3.5"
   - "3.6"
-install:
-  - pip install coveralls coverage
-  - pip install $DJANGO
-script:
-  - coverage run runtests.py
-  - coverage report -m
-after_script:
-  - coverage combine
-  - coveralls
+  - "3.7"
+
+sudo: false
+
 env:
-  - DJANGO="Django>=1.4,<1.5"
-  - DJANGO="Django>=1.5,<1.6"
-  - DJANGO="Django>=1.6,<1.7"
-  - DJANGO="Django>=1.7,<1.8"
   - DJANGO="Django>=1.8,<1.9"
   - DJANGO="Django>=1.9,<1.10"
   - DJANGO="Django>=1.10,<1.11"
   - DJANGO="Django>=1.11,<2.0"
+  - DJANGO="Django>=2.0,<2.1"
```

Comment:
Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.